cloudvolume/meshservice.py

```diff
@@ -110,8 +142,13 @@ def produce_output(mdata):
       ])
 
       if remove_duplicate_vertices:
-        vertices, faces = np.unique(vertices[faces], return_inverse=True, axis=0)
-        faces = faces.astype(np.uint32)
+        if chunk_size:
+          vertices, faces = remove_duplicate_vertices_cross_chunks(vertices,
```

Comment:
What happens if you specify `remove_duplicate_vertices` but not `chunk_size`? Maybe remove_duplicate_vertices could be a boolean (in which case it removes all duplicate vertices) or a tuple (in which case it removes all fragment overlaps).

Reply:
Huh, I must have read the indentation wrong.